<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="te_lylic">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>371</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_push">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>270</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string>확인</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_result">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>340</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>장르</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_result_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>380</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>퍼센트</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
